Automattic/wp-calypso
# Help Centre: Decode Entities on Post Title

## Proposed Changes
Simple fix to decode entities on the post title for the Help Centre.

## Why are these changes being made?
The post content already does this in the same way, so I think it was just an oversight not to include the function on the title as well. 

## Testing Instructions

* Go to My Home and scroll to the "Get Help" section
* Search for the "Top Posts & Pages Block" article (or just any article with such symbols)
* Confirm the title now appears correctly

| Before | After |
|--------|--------|
| <img width="411" alt="Screenshot 2024-08-20 at 16 12 00" src="https://github.com/user-attachments/assets/11053851-9317-4c6b-9f9f-9a8a7ee8c8f5"> | <img width="424" alt="Screenshot 2024-08-20 at 16 11 54" src="https://github.com/user-attachments/assets/c7bf3583-15de-4dd7-a73f-3eabef6cb490"> | 

cc @arcangelini, @alshakero
